{
  "Header": {
    "Original": "CVE-2014-0482 (The contrib.auth.middleware.RemoteUserMiddleware middleware in Django  ...)",
    "Line": 473747,
    "ID": "CVE-2014-0482",
    "Description": "(The contrib.auth.middleware.RemoteUserMiddleware middleware in Django  ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-3010-1 DLA-65-1}",
      "Line": 473748,
      "Type": "xref",
      "Bugs": [
        "DSA-3010-1",
        "DLA-65-1"
      ]
    },
    {
      "Original": "- python-django 1.6.6-1",
      "Line": 473749,
      "Type": "package",
      "Package": "python-django",
      "Kind": "fixed",
      "Version": "1.6.6-1"
    }
  ]
}